File: MWR/Source/MWRSolution/MWRDBModule/DBModule/TblMWInvAuthorizeAttach.cs
using System;
using System.Collections.Generic;
using System.Text;
using ComLib.db;

namespace YRKJ.MWR
{

    public class TblMWInvAuthorizeAttach : BaseDataModule
    {

        private static string TableName = "MWInvAuthorizeAttach";
        public TblMWInvAuthorizeAttach()
        {
        }
        public static string getFormatTableName()
        {
            return SqlCommonFn.FormatSqlTableNameString(TableName);
        }

        public static DataColumnInfo[] Columns = 
                new DataColumnInfo[]{
            new DataColumnInfo(true,false,false,false,"InvAttachId",SqlCommonFn.DataColumnType.INT,10),
            new DataColumnInfo(false,true,false,false,"InvAuthId",SqlCommonFn.DataColumnType.INT,10),
            new DataColumnInfo(false,true,false,false,"Path",SqlCommonFn.DataColumnType.STRING,128)
        };

        public static DataColumnInfo getInvAttachIdColumn()
        {
            return Columns[0];
        }
        public static DataColumnInfo getInvAuthIdColumn()
        {
            return Columns[1];
        }
        public static DataColumnInfo getPathColumn()
        {
            return Columns[2];
        }

        private int _InvAttachId = 0;
        private int _InvAuthId = 0;
        private string _Path = "";

        public int InvAttachId
        {
            get
            {
                return _InvAttachId;
            }
            set
            {
                _InvAttachId = value;
            }
        }
        public int InvAuthId
        {
            get
            {
                return _InvAuthId;
            }
            set
            {
                _InvAuthId = value;
            }
        }
        public string Path
        {
            get
            {
                return _Path;
            }
            set
            {
                _Path = value;
            }
        }

         public override void SetValue(System.Data.DataRow row)
         {
              _dataRow = row;
             System.Data.DataColumnCollection dataCols = row.Table.Columns;
             if(dataCols.Contains("InvAttachId"))
                 SetValue(ref _InvAttachId, row["InvAttachId"]);
             if(dataCols.Contains("InvAuthId"))
                 SetValue(ref _InvAuthId, row["InvAuthId"]);
             if(dataCols.Contains("Path"))
                 SetValue(ref _Path, row["Path"]);
             if(dataCols.Contains("TEM_COLUMN_COUNT"))
                 SetValue(ref _TEM_COLUMN_COUNT, row["TEM_COLUMN_COUNT"]);
         }


    }
}

File: DemoApp/TblModel/TblMWInvAuthorizeAttach.java
package com.yrkj.ar.bean;
import com.yrkj.lib.db.BaseDataModule;
import com.yrkj.lib.db.DataColumnInfo;
import com.yrkj.lib.db.SqlCommonFn;
import com.yrkj.lib.db.SqlCommonFn.DataColumnType;

public class TblMWInvAuthorizeAttach extends BaseDataModule {
    private static String TableName = "MWInvAuthorizeAttach";
    public TblMWInvAuthorizeAttach(){
    }
    public static String getFormatTableName(){
        return SqlCommonFn.FormatSqlTableNameString(TableName);
    }

    public static DataColumnInfo[] Columns = 
            new DataColumnInfo[]{
        new DataColumnInfo(true,false,false,false,"InvAttachId",DataColumnType.INT,10),
        new DataColumnInfo(false,true,false,false,"InvAuthId",DataColumnType.INT,10),
        new DataColumnInfo(false,true,false,false,"Path",DataColumnType.STRING,128)
    };

    public static DataColumnInfo getInvAttachIdColumn(){
        return Columns[0];
    }
    public static DataColumnInfo getInvAuthIdColumn(){
        return Columns[1];
    }
    public static DataColumnInfo getPathColumn(){
        return Columns[2];
    }

    public int InvAttachId = 0;
    public int InvAuthId = 0;
    public String Path = "";

    public int getInvAttachId() {
        return  InvAttachId;
    }
    public void setInvAttachId(int InvAttachId) {
        this.InvAttachId = InvAttachId;
    }
    public int getInvAuthId() {
        return  InvAuthId;
    }
    public void setInvAuthId(int InvAuthId) {
        this.InvAuthId = InvAuthId;
    }
    public String getPath() {
        return  Path;
    }
    public void setPath(String Path) {
        this.Path = Path;
    }



}
